src/components/TodoList.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import TodoForm from "./TodoForm";
+import Todo from "./Todo";
+import { ColorSchemeProvider, MantineProvider, ActionIcon, Group, Title, Center } from '@mantine/core'
+import { MoonStars, Sun } from 'tabler-icons-react';
+
+import { useLocalStorage } from "@mantine/hooks";
+
+function TodoList() {
+  const [todos, setTodos] = useState([
+    {
+      text: "Learn about React",
+      id: 1,
+      isCompleted: false,
+    },
+    {
+      text: "Meet friend for lunch",
+      id: 2,
+      isCompleted: false,
+    },
+    {
+      text: "Build really cool todo app",
+      id: 3,
+      isCompleted: false,
+    },
+  ]);
+
+	const [colorScheme, setColorScheme] = useLocalStorage({
+		key: 'mantine-color-scheme',
+		defaultValue: 'dark',
+		getInitialValueInEffect: true,
+	});
+	const toggleColorScheme = () => {
+		setColorScheme(colorScheme === 'dark' ? 'light' : 'dark');
+    console.log(colorScheme);
+  }
+
+  //
+  const addTodo = (todo) => {
+    const newTodos = [todo, ...todos];
+    console.log(newTodos);
+    setTodos(newTodos);
+  };
+
+  const updateTodos = (todoId, newValue) => {
+    console.log(newValue);
+    console.log(todoId);
+    console.log(todos);
+    newValue.id = todoId.id;
+    // if (!newValue || /^|s*$/.test(newValue.text)) {
+    //   return;
+    // }
+    setTodos((prev) =>
+      prev.map((item) => (item.id === todoId.id ? newValue : item))
+    );
+  };
+
+  const removeTodo = (id) => {
+    const removeArr = [...todos].filter((todo) => todo.id !== id);
```

Comment:
```suggestion
    const removeArr = todos.filter((todo) => todo.id !== id);
```

making a copy of todos here is redundant, as filter will return a new array anyways